### DISCUSSION TITLE
Documentation needs

### DISCUSSION THREAD

**Discussion by melchiork** [reqnroll/Reqnroll#281] [Contributor Q&A]
Hi Folks,

I'd be happy to work on documentation - what parts do you suggest need most attention?

**Comment by gasparnagy** [comment]
Thx! Probably at the IDE integration part: https://docs.reqnroll.net/latest/ide-integrations/index.html but having a general review and fix typos would be also useful.

**Reply by melchiork** [reply]
There are already xsrt files in the specific IDE plugins folders. I never worked with xsrt before, should those be retained or rewritten to markdown?
Basically should I amend existing SpecFlow and use markdown format docs or write new docs from scratch?

**Reply by gasparnagy** [reply]
I'm also not experienced with xsrt, so the best would be to rewrite it to markdown. Thx!
